src/midelware/globalErrorsMiddleware.js

```diff
@@ -0,0 +1,12 @@
+import { generalLogger as logger } from '../logger/logger'
+import { StatusCodes } from 'http-status-codes'
+import { INTERNAL_SERVER_ERROR } from '../error/errorMessages'
+
+const ERROR_MESSAGE = 'Error occurred.'
+
+function handleError (err, req, res) {
+  logger.error(ERROR_MESSAGE, err)
```

Comment:
Should we log here: method name, arguments(url), error message due to **TASK 5.3** requirements?

Reply:
Yes, this is done in src/logger/logger.js configs

![image](https://user-images.githubusercontent.com/37336437/146819900-4402d838-a736-4de5-ac6f-3a5d56514920.png)

![image](https://user-images.githubusercontent.com/37336437/146820126-2e5d7419-2795-4d1d-9f79-8147b6ea8d37.png)